app/controllers/pages_controller.rb

```diff
@@ -44,4 +44,8 @@ def news
     @latestPatchNote = PatchNote.last
     @latestPatchEntries = PatchEntry.where('patchnote_id = ?', @latestPatchNote.id).order('importance DESC')
   end
+
+  def moderation
+    @moderator_logs = ModeratorLog.order('created_at DESC').limit(50)
+  end
```

Comment:
I think it might be worth filtering out approvals for new users' submissions (as opposed to re-approving a previously soft-removed submission) because it's basically a non-action and would spam up the moderator log *fast* compared to the stuff that people care about like removed posts or bans (potentially dozens of actions per day!)